packages/rum-core/test/performance-monitoring/capture-hard-navigation.spec.js

```diff
@@ -199,16 +200,14 @@ describe('Capture hard navigation', function() {
   })
 
   it('should captureHardNavigation', function() {
-    var tr = new Transaction('test', 'test')
-    tr.isHardNavigation = true
+    var tr = new Transaction('test', PAGE_LOAD)
```

Comment:
that could totally ruin the transaction dashboard. span timings wont match with the transaction start and it would simply be a `noop`, which means the users wont be able to see any spans on the UI.